internal/core/integrations/gitlab_integration.go

```diff
@@ -512,8 +513,14 @@ func (g *gitlabIntegration) addProjectHook(ctx core.Context) error {
 		return fmt.Errorf("could not list project hooks: %w", err)
 	}
 
+	chosenURL := os.Getenv("INSTANCE_DOMAIN")
+	if chosenURL == "" {
+		slog.Error("No URL specified in .env file")
+		return nil
+	}
+
 	for _, hook := range hooks {
-		if hook.URL == "https://api.main.devguard.org/api/v1/webhook/" {
+		if hook.URL == chosenURL {
```

Comment:
I prefer passing the domain without any path. Like: "https://api.main.devguard.org"

```suggestion
		if strings.HasPrefix(hook.URL, chosenURL) {
```